src/components/main/CardMovie.tsx

```diff
@@ -69,10 +53,10 @@ function CardMovie(props: Props) {
             <MovieMenu setIsOpen={setIsOpenMenu} setIsOpenDeleteModal={setIsOpenDeleteModal} setIsOpenEditModal={setIsOpenEditModal} />
           ) }
         </div>
-        <img src={movie.image} alt={`${movie.title}.png`} className={styles.movieImage} onClick={() => selectMovie(movie)} />
+        <img src={movie.poster_path} alt={`${movie.title}.png`} className={styles.movieImage} onClick={() => selectMovie(movie)} />
```

Comment:
I guess that there is no much sense to add `png` in alt:
```suggestion
        <img src={movie.poster_path} alt={movie.title} className={styles.movieImage} onClick={() => selectMovie(movie)} />
```

Reply:
resolved